liquibase-standard/src/main/java/liquibase/GlobalConfiguration.java

```diff
@@ -261,6 +263,10 @@ public class GlobalConfiguration implements AutoloadedConfigurations {
                 .setDescription("If true 'classpath:' prefix will be preserved in normalized paths, allowing to resolve hierarchical resources under a classpath-based root.")
                 .setDefaultValue(false)
                 .build();
+        IGNORE_MISSING_REFERENCES = builder.define("diff.ignoreMissingReferences", Boolean.class)
```

Comment:
Could this be moved to `DiffCommandStep` and set up in `ObjectUtil` around there? Well.. this would be a static set, so not sure how it would improve ... just a thought.

Reply:
Let me review this again.

Reply:
The arguments are now command line arguments.  Thanks @filipelautert for the suggestion.